components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/action/model/OperationExecutionResult.java

```diff
@@ -0,0 +1,44 @@
+package org.wso2.carbon.identity.oauth.action.model;
+
+import org.wso2.carbon.identity.action.execution.model.PerformableOperation;
+
+/**
+ * This class represents the result of the execution of an operation.
+ * It contains the operation that was executed, the status of the execution and a message.
+ * This is used to summarize the operations performed based on action response.
+ */
+public class OperationExecutionResult {
```

Comment:
Since this is generic class please check whether we can move this class to the event core and reuse for future implementations

Reply:
Agree. Will move as we progress for one or two more extensions. It's fine to break these contracts as we are still under development